### PR TITLE
[batch] billing project ui

### PR DESCRIPTION
**Pull request by cseed**
Added simple UI for billing projects, create, add and remove users.  Hand-tested with dev deploy.

### CODE REVIEW

File: batch/batch/front_end/front_end.py [hunk 1]
```diff
@@ -829,6 +833,146 @@
     return await render_template('batch', request, userdata, 'job.html', page_context)
 
 
+@routes.get('/billing_projects')
+@prom_async_time(REQUEST_TIME_GET_BILLING_PROJECTS_UI)
+@web_authenticated_users_only()
```
Comment (danking) [inline batch/batch/front_end/front_end.py]: dev only? Are we punting on locking down this stuff further?

Comment (cseed) [reply batch/batch/front_end/front_end.py]: Further, how?  Right now we don't have finer grain permissions or other classes of users.

I've thought about adding an admin role to the users table.  Which dev wouldn't be an admin at this point?

Eventually we're going to want finer grain permissions like a billing creator role who can create billing projects, and a billing manager who can add/remove users from specific billing projects.  I imagine Claire and Sam would be billing mangers.

Also, at some point we should let members of the team view user's batches for debugging purposes.  Should any dev be be able to do this?

So I would say yes, but this seems out of scope for this PR.  This is mainly so you don't need to edit the billing project tables by hand in mysql (what I'm currently doing).

**Comment by cseed** [comment]
Rebased and now use the new database transaction logic.

File: batch/batch/front_end/front_end.py (hunk 1 above)
Comment (danking) [reply batch/batch/front_end/front_end.py]: Sorry, I was unclear. Shouldn't this be `@web_authenticated_developers_only()`? AFAICT, none these billing endpoints should be accessible to, for example, arbitrary ATGU folks.

**Comment by cseed** [comment]
Thanks for clarifying, you were 100% right. Fixed.

**Pull request merged**